ihe-interop/src/test/java/org/motechproject/ihe/interop/handler/helper/IHEActionHelperTest.java

```diff
@@ -75,12 +77,10 @@ public void shouldCallSendTemplateMethod() throws IOException, SAXException, Par
         params.put(Constants.TEMPLATE_NAME_PARAM, "sampleTemplate3");
         params.put(Constants.RECIPIENT_NAME_PARAM, "sampleRecipient3");
 
-        byte[] bytes = IOUtils.toByteArray(getClass().getResourceAsStream("/empty_template.xml"));
-        Byte[] byteObjects = new Byte[bytes.length];
-        int i = 0;
-        for (byte b : bytes) {
-            byteObjects[i++] = b;
-        }
+        InputStream inputStream = getClass().getResourceAsStream("/empty_template.xml");
+        byte[] bytes = IOUtils.toByteArray(inputStream);
+        inputStream.close();
```

Comment:
streams should be closed in finally. In java 7 you can do: 

try (InputStream is = ...) {

}

to achieve the same